src/Sylius/Bundle/ShopBundle/templates/shared/icon.html.twig

```diff
@@ -1,5 +1,5 @@
 {% macro icon(iconName, style) %}
-    {% set icon = source('@SyliusShopAssets/icons/' ~ iconName ~ '.svg', ignore_missing = true) %}
```

Comment:
IMO all of these should be treated as assets (which they are), instead of being moved to the templates directory

Reply:
You are right. This is just a temporary fix to allow the end application to work. All this icon-related code will be removed, and we will use https://ux.symfony.com/icons instead.